src/Forms/SudoModePasswordField.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Forms\FormField;
+
+class SudoModePasswordField extends FormField
+{
+    public const FIELD_NAME = 'SudoModePasswordField';
+    
+    protected $schemaComponent = 'SudoModePasswordField';
+
+    protected $extraClasses = [
+        'SudoModePasswordField',
+    ];
```

Comment:
The existing pattern is to call `$this->addExtraClass()` in the constructor instead.
```suggestion
```

Reply:
Updated